.code-samples.meilisearch.yaml

```diff
@@ -6,7 +6,7 @@
 get_one_index_1: |-
   $client->index('movies')->fetchRawInfo();
 list_all_indexes_1: |-
-  $client->getAllIndexes();
+  $client->getAllIndexes((new IndexesQuery())->setFields(['id', 'title', 'poster', 'release_date']));
```

Comment:
I've updated it, but I saw `setFields` does not exist yet. Is it planned to add it?

Reply:
I was wrong! Solving this!